sympy/solvers/tests/test_solveset.py

```diff
@@ -931,6 +931,8 @@ def test_solveset():
                                                   S.Integers)
     # issue 13825
     assert solveset(x**2 + f(0) + 1, x) == {-sqrt(-f(0) - 1), sqrt(-f(0) - 1)}
+    assert solveset(x*Max(x, 15) - 10, x) == FiniteSet(S("2/3"))                                #issue 10158
```

Comment:
as in the case immediately above, just put the reference number on its own line

Reply:
Sure!